dnd5e.mjs

```diff
@@ -320,6 +313,13 @@ function _configureStatusEffects() {
  * Prepare attribute lists.
  */
 Hooks.once("setup", function() {
+  // Configure trackable & consumable attributes.
+  _configureTrackableAttributes();
+  _configureConsumableAttributes();
+
+  // Set up status effects
+  _configureStatusEffects();
```

Comment:
This would prevent users from pushing to `CONFIG.statusEffects` on the `init` hook unless they add their new status to `CONFIG.DND5E.statusEffects`.

Reply:
Good point. Neither is a good option as `CONFIG.DND5E.statusEffects` is probably the right place to add for the system based on what it's doing, but not for everyone currently pushing to `CONFIG.statusEffects`. Equally, having it in the `init` means subsequently adding into `CONFIG.DND5E.statusEffects` isn't reflected in `CONFIG.statusEffects`. I'll have a look to see what instances are using `CONFIG.statusEffects` to see if it's an issue like with trackable attributes.

Reply:
Looks like `CONFIG.DND5E.statusEffects` is only used in this instance to populate `CONFIG.statusEffects`, so the answer for users/modules seems to be to push to `CONFIG.statusEffects` instead of `CONFIG.DND5E.statusEffects`. I'll move `_configureStatusEffects` back to the `init`.